integration/templates/chart_operator_values.go

```diff
@@ -5,6 +5,8 @@ package templates
 // ChartOperatorValues values required by chart-operator-chart.
 const ChartOperatorValues = `
 clusterDNSIP: 10.96.0.10
+e2e: true
```

Comment:
This is so the `giantswarm-critical` priority class is created. Otherwise chart-operator won't start and e2e fails.